book/src/developers/contributing/rotation/index.md

```diff
@@ -66,22 +90,15 @@ When you get to the end of your checklist, here are ideas for what to work on ne
 - Add new integration tools (eg~ portal-hive -> Discord bot?)
 - Scratch your own itch! What do you wish was easier? Now is the time to build it.
 
-Be sure to update the week's notes as you go, adding: notable incidents, summaries of what you are working on, and ideas for future work.
-
 ### Handoff
 
-Before handing off the notes on Friday, be sure to read through the notes to cleanly summarize the week's activities. Link to any helpful issues, and delete any text that is only meaningful to you.
+Look back at your week as Flamingo and summarize it in notes if needed. Prepare for a kickoff discussion with the next Flamingo and update them on your work from previous week.
 
-Double check the [Flamingo schedule](https://notes.ethereum.org/@njgheorghita/r1angO2lT) and make sure you're available for your next rotation. If not, please switch with somebody asap.
-
-Think through: what kinds of things do you wish were on the notes that you got on Monday?
+Especially while we continue to develop the procedure, try to be available the following Monday to help the subsequent Flamingo transition in, and get started.
 
-After a while, everyone should have the Flamingo Notes link. Sending the link anyway helps accomplish a few things:
-1. Signal to the next Flamingo that you're done writing up the notes
-2. Give the next Flamingo a reminder that their turn is coming up
-3. A failsafe to make sure we didn't forget to pick a Flamingo for the following week
+Think through: what kinds of things do you think should on the checklist?
 
-Especially while we continue to develop the procedure, try to be available the following Monday to help the subsequent Flamingo transition in, and get started.
+Double check the [Flamingo schedule](https://notes.ethereum.org/@njgheorghita/r1angO2lT) and make sure you're available for your next rotation. If not, please switch with somebody asap.
```

Comment:
This is also another remnant that isn't actively maintained. 
- I can update the schedule
- Update this to say something like... "estimate when you'll be flamingo next and make sure you're available..."

Reply:
As I said in the other comment, I think we should restart using it.

And there shouldn't be "estimation" on when next shift should come, meaning that it should always be planned at least one-two months in the future.

I volunteer to take responsibility and make sure we always have a schedule ahead. Others just have to be sure they keep it updated when they change shifts.